browser/brave_content_browser_client.cc

```diff
@@ -334,6 +382,12 @@ void BraveContentBrowserClient::RegisterBrowserInterfaceBindersForFrame(
     map->Add<brave_search::mojom::BraveSearchDefault>(
         base::BindRepeating(&BindBraveSearchDefaultHost));
   }
+
+#if BUILDFLAG(BRAVE_ADS_ENABLED)
+  map->Add<brave_ads::mojom::BraveAdsDriver>(
```

Comment:
why aren't we checking `IsRequestAdsEnabledApiEnabled` ?

Reply:
Fixed